geth-utils/gethutil/mpt/trie/stacktrie.go

```diff
@@ -760,11 +737,14 @@ func (st *StackTrie) GetProof(db ethdb.KeyValueReader, key []byte) ([][]byte, er
 			}
 
 		}
-	}
-
-	fmt.Println("----------")
-	for i := 0; i < len(proof); i++ {
-		fmt.Println(proof[i])
+		// The proof is now reversed (only for non-hashed),
+		// let's reverse it back to have the leaf at the bottom:
+		l := len(proof)
+		var proof_sorted [][]byte
+		for i := 0; i < l; i++ {
+			proof_sorted = append(proof_sorted, proof[l-i-1])
+		}
+		proof = proof_sorted
```

Comment:
Golang has a built-in function for that. Can we try function? https://pkg.go.dev/slices@master#Reverse
```suggestion
		proof = slices.Reverse(proof)
```

Reply:
Yes, much better. This was not possible in Go 1.6 though, so I switched to 1.8.